packages/@angular/cli/commands/test.ts

```diff
@@ -14,13 +14,15 @@ export interface TestOptions {
   build?: boolean;
   sourcemap?: boolean;
   progress?: boolean;
+  configFile: string;
 }
 
 
 const TestCommand = EmberTestCommand.extend({
   availableOptions: [
     { name: 'watch', type: Boolean, default: true, aliases: ['w'] },
     { name: 'code-coverage', type: Boolean, default: false, aliases: ['cc'] },
+    { name: 'config-file', type: Boolean, default: false, aliases: ['c', 'cf'] },
```

Comment:
This should probably be `config` with an alias of `c` to be consistent with the upcoming e2e changes. (#4527)